### PR TITLE
Dvasia Peak

### PR DESCRIPTION
**Pull request by vtange**
for some reason don't find any quests in Sunset Flag Forest. I'm level 132. Too low/high?

### CODE REVIEW

File: QUEST_LV_0200.tsv [hunk 1]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
+QUEST_LV_0200_20150317_001950	$Talk to Soldier Carlos
+QUEST_LV_0200_20150317_001951	$You saved Soldier Carlos who was running away from the monsters. Talk yo Soldier Carlos.
+QUEST_LV_0200_20150317_001952	$Defeat the monsters threatening the soldier
+QUEST_LV_0200_20150317_001953	$Talk to the Settled Owl Sculpture
+QUEST_LV_0200_20150317_001954	$There is a hesitant-looking Owl Sculpture. Talk to the Settled Owl Sculpture.
+QUEST_LV_0200_20150317_001955	$Defeat nearby Woodgoblins
+QUEST_LV_0200_20150317_001956	$The Settled Owl Sculpture wants to repay the soldiers who saved it. To help the soldiers, please defeat nearby Ravinelarvae.
+QUEST_LV_0200_20150317_001957	$You defeated all nearby Ravinelarvae. Talk to the Settled Owl Sculpture.
+QUEST_LV_0200_20150317_001958	$Defeat Ravinelarvae
+QUEST_LV_0200_20150317_001959	$Talk to Soldier Daniels
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1959]: Daniels -> Dainus

File: QUEST_LV_0200.tsv [hunk 2]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
+QUEST_LV_0200_20150317_001960	$Soldier Daniels is trembling from fear. To look for the other soldiers, Talk to Daniels.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1960]: Daniels -> Dainus 
To look for the other soldiers, Talk to Daniels. -> Talk to Soldier Dainus to look for the other soldiers.

File: QUEST_LV_0200.tsv [hunk 3]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
+QUEST_LV_0200_20150317_001941	$Defeat Treegools at Byreti Vacant Lot
+QUEST_LV_0200_20150317_001942	$The Substitute Owl Sculpture wants to help out the soldiers. On behalf of the Substitute Owl Sculpture, defeat Treegools at Byreti Vacant Lot.
+QUEST_LV_0200_20150317_001943	$Report to the Substitute Owl Sculpture
+QUEST_LV_0200_20150317_001944	$You've defeated all Treegools you could find. Report to the Substitute Owl Sculpture.
+QUEST_LV_0200_20150317_001945	$Defeat Treegools
+QUEST_LV_0200_20150317_001946	$Talk with O'Rourke
+QUEST_LV_0200_20150317_001947	$Commander O'Rourke is astonished by your skills after seeing you defeating the monster ambush. Talk with Commander O'Rourke.
+QUEST_LV_0200_20150317_001948	$Look for the soldiers at Niurus Highway
+QUEST_LV_0200_20150317_001949	$Commander O'Rourke is saying that his soldiers who were dispatched to an operation have not returned. Go to Niurus Highway and look for the soldiers.
+QUEST_LV_0200_20150317_001950	$Talk to Soldier Carlos
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1950]: It's not Carlos.

It can be: Kalus, Kallus or Callus.

File: QUEST_LV_0200.tsv [hunk 4]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
+QUEST_LV_0200_20150317_001951	$You saved Soldier Carlos who was running away from the monsters. Talk yo Soldier Carlos.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1951]: It's not Carlos.

It can be: Kalus, Kallus or Callus.

File: QUEST_LV_0200.tsv [hunk 5]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
+QUEST_LV_0200_20150317_001952	$Defeat the monsters threatening the soldier
+QUEST_LV_0200_20150317_001953	$Talk to the Settled Owl Sculpture
+QUEST_LV_0200_20150317_001954	$There is a hesitant-looking Owl Sculpture. Talk to the Settled Owl Sculpture.
+QUEST_LV_0200_20150317_001955	$Defeat nearby Woodgoblins
+QUEST_LV_0200_20150317_001956	$The Settled Owl Sculpture wants to repay the soldiers who saved it. To help the soldiers, please defeat nearby Ravinelarvae.
+QUEST_LV_0200_20150317_001957	$You defeated all nearby Ravinelarvae. Talk to the Settled Owl Sculpture.
+QUEST_LV_0200_20150317_001958	$Defeat Ravinelarvae
+QUEST_LV_0200_20150317_001959	$Talk to Soldier Daniels
+QUEST_LV_0200_20150317_001960	$Soldier Daniels is trembling from fear. To look for the other soldiers, Talk to Daniels.
+QUEST_LV_0200_20150317_001961	$Look for the other soldiers in Drieza Waterland.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1961]: in -> at

File: QUEST_LV_0200.tsv [hunk 6]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
+QUEST_LV_0200_20150317_001962	$Soldier Daniels told you that there may be other soldiers on Drieza Waterland. Go to Drieza Waterland and look for the soldiers there.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1962]: on -> at

File: QUEST_LV_0200.tsv [hunk 7]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
-QUEST_LV_0200_20150317_001960	The soldier Dainews is trembling from the fear. To look for the other soldiers, Talk with Dainews.
-QUEST_LV_0200_20150317_001961	Look for the other soldiers in Drieza dried fields.
-QUEST_LV_0200_20150317_001962	Soldier Danius told you that there may be other soldiers on Drieza dried fields. Go to Drieza dried fields and look for the soldiers there.
-QUEST_LV_0200_20150317_001963	Defeat ronbaum
+QUEST_LV_0200_20150317_001923	$Defeat the Giant Redwood
+QUEST_LV_0200_20150317_001924	$Talk to Commander Julian
+QUEST_LV_0200_20150317_001925	$Commander Julian is troubled with no one able to do a mission. Talk to Commander Julian.
+QUEST_LV_0200_20150317_001926	$Hand it over to Soldier Paulter
+QUEST_LV_0200_20150317_001927	$Commander Julian asked you to help with his work. Pass the order he gave you to the Soldier Paulter at Zaizda Hill.
+QUEST_LV_0200_20150317_001928	$Talk to Soldier Paulter
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1928]: Paulter is incorrect.

The name can either be: Polter, Poulter.

File: QUEST_LV_0200.tsv [hunk 8]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
+QUEST_LV_0200_20150317_001929	$Upon receiving the order, Paultner's face grew gloomy. Talk to Soldier Paulter again.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1929]: 'Paultner' - Also, see above note about the name. (I'm not gonna leave a note on the other lines)

File: QUEST_LV_0200.tsv [hunk 9]
```diff
@@ -1893,74 +1893,74 @@ QUEST_LV_0200_20150317_001892	I can get them
...
-QUEST_LV_0200_20150317_001913	Even after you brought the dried branches, it seems that Samsom's worry doesn't go away. Talk with Soldier Samson again.
-QUEST_LV_0200_20150317_001914	Set the firewoods on the top of Ishisineti Cliffs.
-QUEST_LV_0200_20150317_001915	It seems that when you burn the disgusting lump by lighting up the firewoods, you may be able to pull out the ring. On behalf of Samson who is concentrating on his work, set the firewoods on the top of the cliffs.
-QUEST_LV_0200_20150317_001916	You are now ready to set the fire. Go back to the Soldier Samson.
-QUEST_LV_0200_20150317_001917	Look into the rotten stump of the tree
-QUEST_LV_0200_20150317_001918	There is a suspiciously looking rotten stump of the tree. How about you touch it?
-QUEST_LV_0200_20150317_001919	Defeat the gigantic Wood Goblin
-QUEST_LV_0200_20150317_001920	Defeat the Wood Goblin that appeared from Suvyte Cliffs.
+QUEST_LV_0200_20150317_001896	$Talk to Soldier Dominique
+QUEST_LV_0200_20150317_001897	$I met with the Soldier Dominique who seems to be puzzled. Talk with Dominique.
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:1897]: 'I met with the' 

Talk with Dominique -> Talk to her.  (?)

File: QUEST_LV_0200.tsv [hunk 10]
```diff
@@ -32,54 +32,54 @@ QUEST_LV_0200_20150317_000031	$The Commander was also struggling like this..{nl}
...
+QUEST_LV_0200_20150317_000064	$It would be nice if more people could come and go using this road like before.{nl}I find people very interesting.
+QUEST_LV_0200_20150317_000065	$Thank you. I hope I'm not giving you needless trouble all because of my aspirations.
+QUEST_LV_0200_20150317_000066	$Alright. We haven't received any word from our forces at Niurus Highway.{nl}I request you find them.
+QUEST_LV_0200_20150317_000067	$If you can't find them, return to us.
+QUEST_LV_0200_20150317_000068	$Soldier Carlos
+QUEST_LV_0200_20150317_000069	$There are still a number of our colleagues further in.{nl}Please help us.
+QUEST_LV_0200_20150317_000070	$Soldier Daniels
+QUEST_LV_0200_20150317_000071	$Ironbaum attacked us at the Drieza Waterland.{nl}There are still soldiers that have not yet escaped.{nl}
+QUEST_LV_0200_20150317_000072	$I'm embarrassed to have escaped by myself.{nl}
+QUEST_LV_0200_20150317_000073	$Settled Owl Statue
```
Comment (Sunaries) [inline QUEST_LV_0200.tsv:73]: Why did you change this one to Statue? The rest of are "Sculpture".

**Comment by Sunaries** [comment]
There are 2 NPCs.
- 장교 다뉴스 (Officer Danus)
- 병사 다이뉴스 (Soldier Dainus)

Currently, 'Officer Danus' is 'Commander Dainus', which is wrong. I've fixed it already, but I haven't pushed the commit yet. I'll wait for IMC to merge your PRs.

**Comment by Sunaries** [comment]
Good to merge

**Comment by imcgames** [comment]
Good :+1:
